Mu2eG4/geom/calorimeter_CsI_v1.txt

```diff
@@ -83,7 +83,7 @@ double calorimeter.crateTThickness            = 5;     // top panel thickness
 double calorimeter.crateSThickness            = 8;     // side panel thickness
 double calorimeter.crateFShieldYLength        = 160;
 double calorimeter.crateFShieldDeltaZ         = 102;
-double calorimeter.FEBToDiskZOffset           = 31.98; // distance to front shield to front disk panel
+double calorimeter.FEBToDiskZOffset           = 1.98; // distance to front shield to front disk panel
```

Comment:
These are changes by 30mm. Do you understand why the validation cdepth plot showed a 10mm shift?